enhancements/console/dynamic-plugins.md

```diff
@@ -185,63 +187,122 @@ extension points will be available.
 
 ```json
 {
-  "$schema": "/path/to/schema/manifest.json",
-  "id": "my-plugin",
+  "name": "my-plugin",
   "version": "1.2.3",
-  "pluginAPI": "4.7.x",
   "displayName": "My Plugin",
+  "dependencies": {
+    "@console/pluginAPI": "*"
+  },
   "extensions": [
     {
       "type": "console.flag",
       "properties": {
-        "handler": {
-          "$codeRef": "example.testHandler"
-        }
+        "handler": { "$codeRef": "example.testHandler" }
       }
     },
     {
       "type": "console.flag/model",
       "properties": {
         "flag": "EXAMPLE",
-        "model": {
-          "group": "example.com",
-          "version": "v1",
-          "kind": "ExampleModel"
-        }
+        "model": { "group": "example.com", "version": "v1", "kind": "ExampleModel" }
       }
     }
   ]
 }
 ```
 
-An extension may contain code references, encoded as object literal
+An extension may contain code references, encoded as object literals
 `{ $codeRef: string }`. The value of `$codeRef` is `moduleName.exportName` or
 `moduleName` (equivalent to `moduleName.default`). Webpack itself will resolve
 the remote module when console calls `container.get` on the
 [remote container](https://webpack.js.org/concepts/module-federation/#dynamic-remote-containers).
 
 #### Plugin Manifest Properties
 
-| Property | Description |
-| ----- | -------- |
-| `name: string` | Used as a unique identifier. Each plugin must have a unique name. |
-| `version: string` | The version of the plugin. Version must be parsable by node-semver, eg. “1.2.3” |
-| `pluginAPI: string` | Semver range of compatible OpenShift console versions. |
-| `displayName?: string` | User friendly display name. |
-| `description?: string` | The description of the plugin. |
-| `extensions: Extension[]` | List of extensions contributed by the plugin. |
-| `dependencies?: { [pluginName: string]: string }` | A plugin which depends on other plugins must explicitly define their dependencies. The key of the map is the plugin name and the value is a semver version range. The plugin is not loaded if a dependency is not met. |
+<table>
```

Comment:
Using HTML table since Markdown-specific table syntax does not support multi-line code blocks.

If there's a better way to do this, please let me know.